native/common/jp_convert.cpp

```diff
@@ -385,6 +436,31 @@ jconverter getConverter(const char* from, int itemsize, const char* to)
 				case 'd': return &Convert<double>::toD;
 			}
 			break;
+		case 'e':
+			if (reverse) switch (to[0])
```

Comment:
I am not sure if there is every a case in which a reversed byte order half floating value would be converted.   Through the order of this operation is reasonable to define.   Reverse the bytes, promote to float then convert to Java type.